### PR TITLE
Node1819plus

### PR DESCRIPTION
**Pull request by Amndeep7**

### CODE REVIEW

File: .github/workflows/draft-release.yml [hunk 1]
```diff
@@ -24,18 +24,18 @@ jobs:
       # write permission is required for autolabeler
       # otherwise, read permission is required at least
       pull-requests: write
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```
Comment (georgedias) [inline .github/workflows/draft-release.yml:27]: Is there an issue if we use latest! or do we want to force to use ubuntu 22.04 for the draft release?

**Review by georgedias — COMMENTED** [review]
Just one question on the draft-release.yml

File: .github/workflows/draft-release.yml (hunk 1 above)
Comment (Amndeep7) [reply .github/workflows/draft-release.yml:27]: We're using ubuntu 22.04 for all the rest of the actions.

**Comment by Amndeep7** [comment]
The reason why we are using Ubuntu 22.04 vs ubuntu-latest is that we ran into versioning issues in the past with the tools that were provided in the image.  If we hardlock on a particular OS version then we can test each updated image for the runner as they show up instead of eating it when we try to do a release or something.

@em-c-rod @georgedias

**Pull request merged**